Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿Random rndm = new Random();
+
+int value = rndm.Next(-20, 20); //Можно ли создать рандом sbyte?
+int value1 = rndm.Next(-20, 20);
+int value2 = rndm.Next(-20, 20);
+int value3 = rndm.Next(-20, 20);
+int value4 = rndm.Next(-20, 20);
+int value5 = rndm.Next(-20, 20);
+int value6 = rndm.Next(-20, 20);
+
+int[] numbers = new int[] { value, value1, value2, value3, value4, value5, value6 };
+var str = string.Join(" ", numbers);
+
+Console.Write("Заданный массив:");
+Console.WriteLine(str);
+
+int temp; //сортировка по возрастанию
+for (int i = 0; i < numbers.Length; i++) // выборка элемента с которым будем сравнивать

```

Comment:
мини оптимизация: последний элемент тут брать не надо, т.к. с ним сравнение было предпоследнего (ниже по коду `k=i+1`)

```suggestion
for (int i = 0; i < numbers.Length - 1; i++) // выборка элемента с которым будем сравнивать

```